src/components/dar_collection_table/CollectionConfirmationModal.js

```diff
@@ -61,6 +66,16 @@ export default function CollectionConfirmationModal(props) {
     onConfirm: openOnClick,
   });
 
+  const deleteModal = h(ConfirmationModal, {
+    showConfirmation,
+    styleOverride: { height: '35%' },
+    closeConfirmation: () => setShowConfirmation(false),
+    title: 'Delete DAR Collection Draft',
+    message: `Are you sure you want to delete ${collection.darCode}?`,
+    header: getModalHeader,
```

Comment:
```suggestion
    header: getModalHeader(),
```

https://github.com/DataBiosphere/duos-ui/pull/1581 should have changed the other modals to include the parentheses as well. Without them, no header is included in the modal. I don't think the `styleOverride` is needed either